Add Italian locale

mrkrash opened the pull request:
I followed the document for translations and added Italian

amugofjava commented:
Hi @mrkrash,

This is great - thanks. One thing I have noticed is that, in the Podcast Index category definitions I can't see the first one: After-Shows. Is it missing? Also, some of the entries are also all in lower case and it would be more consistent if they all started upper case.

It's a bit tricky, the iTunes and Podcast Index categories. The APIs only provide categories in English, so I use these CSV Strings as a way to map the English to the other language, but for it to work correctly they have to be in the same order and use the same case.

mrkrash commented:
Hi @amugofjava, I have fixed lowercase and categories.
For test the translation on emulator, for recognize new locale, in my local copy, I have also modified other files but I don't know whether I should include them in this PR.
I have rebased my local copy and now, in this PR, there are also your commits. I hope this is not a problem. Otherwise I clean everything up and re-propose the PR.

amugofjava commented:
Hi @mrkrash,

I see you have added additional translations for the recent changes I have made -  thank you for that. I would say that, as you have translated it all you can also update the other files as well. This way, the PR contains the complete set of code to support Italian and authored by you.

I would prefer a standard merge rather than a rebase. I haven't done a lot of rebasing (I still have plenty of Git to learn), but it can be problematic for other users as a rebase rewrites commit history and so their history will no longer match that of the pushed rebase.

amugofjava commented:
This is great. Thank you @mrkrash for this, it's much appreciated. One tip that I learned recently is, if you start your commit comment with the issue number, so in this case #122 it will link your changes to that issue.
